package.json

```diff
@@ -37,7 +37,7 @@
     "@alfresco/adf-core": "6.4.0-6550080792",
     "@alfresco/adf-extensions": "6.4.0-6550080792",
     "@alfresco/eslint-plugin-eslint-angular": "6.4.0-6550080792",
-    "@alfresco/js-api": "7.1.0-1409",
+    "@alfresco/js-api": "7.1.0-1437",
```

Comment:
Only JS-API version has been bumped, will run upstream once again to check if it will update ADF too, if not we will have to create separate PR for that